src/components/pages/SingleEventPage.tsx

```diff
@@ -105,9 +108,7 @@ class SingleEventPage extends React.Component<Props, State> {
       return null;
     } else {
       return (
-        <div>
-          <img className={classes.image_styles} src={fetchedContent.images[0].url} alt=""></img>
-        </div>
+        <img className={classes.image_styles} src={fetchedContent.images[0].url} alt=""></img>
```

Comment:
There should be default picture, if event has no image ( https://mantyharju.fi/tapahtumat )

Reply:
now the element is completely hidden if there is no image - I think no default image is needed